app/models/user.rb

```diff
@@ -3,7 +3,8 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
-  has_many :metrics
+  
+  has_many :metrics, through: :features
```

Comment:
@dcox Is this has-many-through association working ok? E.g. try in the rails console: `User.first.metrics`